torch/tensor.py

```diff
@@ -64,6 +64,17 @@ def __reduce_ex__(self, proto):
                     self.requires_grad,
                     OrderedDict())
             return (torch._utils._rebuild_qtensor, args)
+        elif self.is_sparse:
+            layout = str(self.layout).split('.', 1)[1]
```

Comment:
Ick! Can't you just directly do an equality test `self.layout == torch.sparse_coo`?

Reply:
The problem is that `torch.sparse_coo` cannot be serialized (unless we implement it also).

Reply:
That seems like a good thing to do. Do you want to land this patch first, or do these together?

Reply:
I'll do it here.